mscore/importgtp-gp6.cpp

```diff
@@ -66,57 +66,151 @@ namespace Ms {
 
 
 const static std::map<QString, QString> instrumentMapping = {
-            {"a-bass4", "acoustic-bass"},
-            {"a-bass5", "acoustic-bass"},
-            {"a-bass6", "acoustic-bass"},
-            {"alt-c",   "alto"},
-            {"alt-s",   "alto"},
-            {"a-piano-gs", "piano"},
-            {"a-piano-ss", "piano"},
-            {"bass-c",     "bass"},
-            {"bass-flt-c", "bass-flute"},
-            {"bassn",      "bassoon"},
-            {"bass-s",     "bass"},
-            {"basstuba-bb",     "bass-eb-tuba"},
-            {"basstuba-eb",     "bb-tuba"},  //???
-            {"bnj4",     "banjo"},
-            {"bnj5",     "banjo"},
-            {"bnj6",     "banjo"},
-            {"cello",    "violoncello"},
-            {"drmkt", "drumset"},
-            {"e-bass4", "bass-guitar"},
-            {"e-bass5", "bass-guitar"},
-            {"e-bass6", "bass-guitar"},
-            {"e-gtr12", "electric-guitar-treble-clef"},
-            {"e-gtr6", "electric-guitar-treble-clef"},
-            {"e-gtr7", "electric-guitar-treble-clef"},
-            {"e-gtr8", "electric-guitar-treble-clef"},
-            {"em-organ-gs", "organ"},
-            {"em-organ-ss", "organ"},
-            {"en-horn", "english-horn"},
-            {"e-piano-gs", "electric-piano"},
-            {"e-piano-ss", "electric-piano"},
-            //{"f-bass5", "bass-guitar"},  /// ??? not in the list
-            {"hrpch-gs", "harpsichord"},
-            {"hrpch-ss", "harpsichord"},
-            {"mrcs", "maracas"},
-            {"mrcs", "oboe"},
-            {"mrcs", "oboe"},
-            {"n-gtr6", "guitar-nylon-treble-clef"},
-            {"n-gtr7", "guitar-nylon-treble-clef"},
-            {"n-gtr8", "guitar-nylon-treble-clef"},
-            {"s-gtr12", "guitar-steel-treble-clef"},
-            {"s-gtr6", "guitar-steel-treble-clef"},
-            {"s-gtr7", "guitar-steel-treble-clef"},
-            {"s-gtr8", "guitar-steel-treble-clef"},
-            {"snt-lead-ss", "poly-synth"},
-            {"tmbrn",  "tambourine"}, // to be mapped
-            {"tnr-s",  "voice"},
-            {"snt-bass-ss", "metallic-synth"},
-            {"vbrslp", "vibraslap"}, // to be mapped
-            {"vla", "viola"},
-            {"vln", "violin"},
-            {"xlphn", "xylophone"}
+            {"2Mrcs",           "maracas"},
```

Comment:
ISTR there was an issue with `2Mrcs`, being a 2 staff instrument (like piano)?

Ah, I see, special case in 2020bcef